frappe/model/mapper.py

```diff
@@ -79,7 +79,7 @@ def get_mapped_doc(
 	):
 		target_doc.raise_no_permission_to("create")
 
-	source_doc = frappe.get_doc(from_doctype, from_docname)
+	source_doc = frappe.get_cached_doc(from_doctype, from_docname)
```

Comment:
This isn't a good idea because we often update docs using SQL query or DB apis which doesn't clear cache. This might cause unnecessary bugs for little benefit. 

Is there any clear benefit for caching here?